### ISSUE TITLE
NOD | Pull email addresses from VA profile for 40 Veterans affected by infinite loop bug

### ISSUE THREAD

**Issue by saderagsdale** [department-of-veterans-affairs/va.gov-team#74252]
## Value Statement
**_As a_** Veteran without a file number in VA's system of record
**_I want to_** have an eFolder created on my behalf using my preferred email address in my profile
**_So that_** I can have letter communications regarding the infinite loop bug added to my file.

---
## Background Context
- The letters that are being generated for veterans affected by the infinite loop bag will be added to their eFfolders. We have identified 40 veterans with no file number available, who need a folder created using their email addresses.

## Acceptance Criteria
- [ ] Email addresses are the most recently added ones from VA profile.

## Designs and Build Notes
- Add here

## Out of scope
- Add here

## Open questions
- Add here

## Tasks
- [x] Open [this sheet](https://dvagov.sharepoint.com/:x:/r/sites/BenefitsDecisionReviewTeam/Shared%20Documents/AH2443211%20Veteran%20and%20POA%20Info%20for%20Notification%20Letter%20V3.xlsx?d=w7c1ec7a388334bf5b68385654345e54c&csf=1&web=1&e=F0vU9b) in VA Sharepoint
- [x] Navigate to the "Results" tab
- [x] Filter to display only users with no file number in column B (named "File Number")
- [x] Search for their preferred email addresses in VA Profile
- [ ] Add the email addresses to a new tab on the sheet; don't edit the "Results" tab

## Definition of Ready 

- [x] Access to sheet
- [x] Instructions on where to place data

## Definition of Done

- [ ] Email addresses for 40 Veterans (or as many as you can find) are added to a separate tab
- [ ] Reviewed and approved by product

**Comment by saderagsdale** [comment]
@data-doge I'll add a link to the list of 40 Veterans who we need email addresses for.

**Comment by saderagsdale** [comment]
@data-doge Added a link to the sheet. This is ready for pickup.

**Comment by HeatherWidmont** [comment]
@data-doge can pull email addresses for most of them, but for about 6 Veterans they will need more information. Eugene will reach out to Cory to see if he can help

**Comment by data-doge** [comment]
Just sent Cory the finalized list of emails, via Teams. Since I don't have access to the sheet (and only have access to a copy of the sheet), I've asked Cory to update the sheet with the emails I provided. 

Wherever possible, I provided the Veterans' most recent email address in VA profile. But some Veterans had no emails there, or had no VA profile record at all. In those cases, I pulled their email address from the `UserCredentialEmail` record associated with their `UserVerification` and `UserAccount` records in VA.gov's database.

I think we can close this ticket now, unless we want to wait until Cory has placed the emails I provided into the sheet.

cc: @saderagsdale
